targets/docker/Dockerfile

```diff
@@ -0,0 +1,69 @@
+# taken from: https://raw.githubusercontent.com/dockerfile/ubuntu/master/Dockerfile
```

Comment:
It looks like the file you used as a base to build on is licensed under the MIT license. This means you have to add the copyright notice and license terms of the original file here.